### ISSUE TITLE
lwn: fix trans error in 682582

### ISSUE THREAD

**Issue by unicornx** [tinyclub/tinylab.org#264]
![1](https://user-images.githubusercontent.com/2476165/53286294-3e8ced00-37a7-11e9-9f73-a70ba82de01c.png)
![2](https://user-images.githubusercontent.com/2476165/53286296-4482ce00-37a7-11e9-9a74-8f0ca7ad0132.png)

**Comment by unicornx** [comment]
according to https://lwn.net/Articles/685236/,
```
Changes since v1
......
- wb_start_writeback() fills in background/reclaim/sync info in
  the writeback work, based on writeback reason.
.....
```
seems that's what 682582 said, `The patch set tries to avoid such unpleasantness by tracking the reason behind each buffered-write operation. `; but since Changes since v3
```
......
- Rewrite blk-wb core as described above, to dynamically adapt. This is
  a big change, see the last patch for a full description of it.
......
```
I believe the core throttling algorithm should have been changed between these two patch version.

Updated 682582 as Xiaohui comments and added commit link to 685894.

**Comment by unicornx** [comment]
merged with pr #265
